src/frontend/screens/Settings/sections/SystemInfo/index.scss

```diff
@@ -0,0 +1,43 @@
+hr {
```

Comment:
let's add a wrapper to nest all these styles to be sure they don't leak to other components, now these styles are not scoped to this screen and some selectors like `.logo` or `hr` are too generic. something like:

```scss
.systeminfo {
  hr {
  }

  .logo {
  }

  ... etc
}
```